### PR TITLE
ding gesellschaft membership page

### PR DESCRIPTION
**Pull request by github-luckow-org**

### CODE REVIEW

File: members/_posts/2017-02-15-dinggesellschaft.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: memberpage
+category: cmember
+logo : gzevd.png
+
+name: 'Ding Gesellschaft'
```
Comment (kaikreuzer) [inline members/_posts/2017-02-15-dinggesellschaft.md]: We would expect your official legal name here, which would rather be "Gesellschaft zur Entwicklung von Dingen mbH", wouldn't it? Or does this break our layout? ;-)

Comment (kaikreuzer) [inline members/_posts/2017-02-15-dinggesellschaft.md]: @github-luckow-org ping!

Comment (github-luckow-org) [inline members/_posts/2017-02-15-dinggesellschaft.md]: pong :)

**Comment by kaikreuzer** [comment]
Merci beaucoup!

**Pull request merged**